src/App.js

```diff
@@ -5,25 +5,22 @@ import './App.scss';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { PostList } from './components/PostList';
+
+const postListWithUsers = [...posts].map(
+  post => Object.assign(
+    post, { user: users.filter(u => u.id === post.userId)[0] },
```

Comment:
```suggestion
    post, { user: users.find(u => u.id === post.userId)[0] },
```
Есть метод для поиска одного элемента

Reply:
done